### DISCUSSION TITLE
What to use in place of deprecated api?

### DISCUSSION THREAD

**Discussion by steve-lorimer** [numToStr/Comment.nvim#233] [Q&A] [answered]
I was previously using the following commands in some custom keybindings:

in normal mode, to toggle the current line:

```
    "<cmd>lua require('Comment.api').toggle_current_linewise()<CR>",
```

in visual mode. to toggle the current block:

```
    ":lua require('Comment.api').toggle_linewise_op(vim.fn.visualmode())<CR>",
```

After upgrading recently, I'm getting an error `... deprecate ...`, obviously signalling the api I was using has since been deprecated.

What should I use instead please?

**Comment by numToStr** [comment, marked as answer]
Replace `toggle_current_linewise` with `toggle.linewise.current` and `toggle_linewise_op` with `toggle.linewise`

**Reply by steve-lorimer** [reply]
Thank you so much! Great plugin btw! :)
